flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -130,6 +130,7 @@ public void close() throws CatalogException {
     }
   }
 
+
```

Comment:
Can we remove this since we don't need to change this file?

Reply:
Removed!